jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2928,3 +2895,51 @@ CHANGELOG=CHANGELOG
 Show\ preview\ in\ entry\ table\ tooltip=Show preview in entry table tooltip
 Citation\ information=Citation information
 Total\ Citations\:\ %0\nSupporting\:\ %1\nContradicting\:\ %2\nMentioning\:\ %3\nUnclassified\:\ %4\nCiting\ Publications\:\ %5=Total Citations: %0\nSupporting: %1\nContradicting: %2\nMentioning: %3\nUnclassified: %4\nCiting Publications: %5
+
+# NewEntry*
+Add\ empty\ entry=Add empty entry
+An\ unknown\ error\ has\ occurred.\nEntries\ may\ need\ to\ be\ added\ manually.=An unknown error has occurred.\nEntries may need to be added manually.
+An\ unknown\ error\ has\ occurred.\nThis\ entry\ may\ need\ to\ be\ added\ manually.=An unknown error has occurred.\nThis entry may need to be added manually.
+Automatically\ determine\ entry\ type=Automatically determine entry type
+Bibliographic\ data\ could\ not\ be\ retrieved.\nThe\ following\ error\ was\ encountered\:\n%0=Bibliographic data could not be retrieved.\nThe following error was encountered:\n%0
+Bibliographic\ data\ could\ not\ be\ retrieved.\nThis\ is\ likely\ due\ to\ an\ issue\ being\ experienced\ by\ the\ server.\nTry\ again\ later.\n%0=Bibliographic data could not be retrieved.\nThis is likely due to an issue being experienced by the server.\nTry again later.\n%0
+Bibliographic\ data\ could\ not\ be\ retrieved.\nThis\ is\ likely\ due\ to\ an\ issue\ with\ your\ input,\ or\ your\ network\ connection.\nCheck\ your\ provided\ identifier\ (and\ identifier\ type),\ and\ try\ again.\n%0=Bibliographic data could not be retrieved.\nThis is likely due to an issue with your input, or your network connection.\nCheck your provided identifier (and identifier type), and try again.\n%0
+Choose\ Entry\ Type=Choose Entry Type
+Create\ new\ entry=Create new entry
+Custom\ types=Custom types
+Enter\ Bib(La)TeX\ sources\ to\ generate\ entries\ from.=Enter Bib(La)TeX sources to generate entries from.
+Enter\ Identifier=Enter Identifier
+Enter\ plain\ citations\ to\ parse,\ separated\ by\ blank\ lines.=Enter plain citations to parse, separated by blank lines.
+Enter\ the\ reference\ identifier\ to\ search\ for.=Enter the reference identifier to search for.
+Failed\ to\ interpret\ citations.\nThe\ following\ error\ was\ encountered\:\n%0=Failed to interpret citations.\nThe following error was encountered:\n%0
+Failed\ to\ interpret\ citations=Failed to interpret citations
+Failed\ to\ lookup\ identifier=Failed to lookup identifier
+Failed\ to\ parse\ Bib(La)Tex=Failed to parse Bib(La)Tex
+Failed\ to\ parse\ entries.\nThe\ following\ error\ was\ encountered\:\n%0=Failed to parse entries.\nThe following error was encountered:\n%0
+Identifier\:=Identifier\:
```

Comment:
Avoid ":" - the labels themselves should state that they are labels - they do not need :